src/main/java/duke/exceptions/InvalidDeadlineException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exceptions;
+
+/**
+ * An exception which indicates that the deadline is invalid.
+ */
+public class InvalidDeadlineException extends Exception{
```

Comment:
Should there be a space between the class declaration and the first bracket ?
I think there are a few more instances.